cmake/cpack.cmake

```diff
@@ -0,0 +1,5 @@
+SET(CPACK_PACKAGE_VERSION_MAJOR 0)
+SET(CPACK_PACKAGE_VERSION_MINOR 1)
+SET(CPACK_PACKAGE_VERSION_PATCH 2)
```

Comment:
[CPACK_PACKAGE_VERSION](https://cmake.org/cmake/help/latest/module/CPack.html#variables-for-advanced-use) gets set in git-versioning.cmake.  That might end up overriding these?

The effect is that the git tag ends up determining the package name. So got nightly you get: `acquire-video-runtime-nightly-win64.7z`

I like setting things based on the git tag, but there might be some good reasons to set it manually. If you do prefer manual, then setting CPACK_PACKAGE_VERSION directly is fine.

Reply:
It doesn't get overridden there, since this is the last cmake file included, and CPack is included right after this. (I learned the hard way that you can't set these variables after including CPack either.)

The reason I set the package version is due to the way the Python build script works. I'm happy to change it to the git tag for runtime and core-libs, but the build script needs that predictable URI in order to fetch the right artifact. The downside is that you can't easily tell the difference between two builds if they're on the same version, but AFAICT the Python build is the only real use case for packaging the dlls right now.